resources/kubernetes/operator/helm/values.yaml

```diff
@@ -184,17 +184,196 @@ collectors:
+                - key: faas.instance
+                - key: faas.name
+                - key: faas.version
+                - key: cloud.provider
+                - key: cloud.region
+                - key: cloud.availability_zone
+                - key: cloud.platform # cloud.servicename
+                - key: cloud.account.id
+              attributes:
+                - key: transaction.root
+                - key: transaction.name
+                - key: transaction.type
+                - key: transaction.result
+                - key: event.outcome
+                - key: metricset.name
+                  default_value: transaction
+                - key: elasticsearch.mapping.hints
+                  default_value: [aggregate_metric_double]
+              unit: us
+              histogram:
+                buckets: [1]
+                value: Microseconds(end_time - start_time)
+            - name: span.destination.service.response_time.sum.us
+              description: APM span destination metrics
+              ephemeral_resource_attribute: true
+              include_resource_attributes:
+                - key: service.name
+                - key: deployment.environment # service.environment
+                - key: telemetry.sdk.language # service.language.name
+                - key: agent.name # set via elastictraceprocessor
+              attributes:
+                - key: span.name
+                - key: event.outcome
+                - key: service.target.type
+                - key: service.target.name
+                - key: span.destination.service.resource
+                - key: metricset.name
+                  default_value: service_destination
+              unit: us
+              sum:
+                value: Double(Microseconds(end_time - start_time))
+            - name: span.destination.service.response_time.count
+              description: APM span destination metrics
+              ephemeral_resource_attribute: true
+              include_resource_attributes:
+                - key: service.name
+                - key: deployment.environment # service.environment
+                - key: telemetry.sdk.language # service.language.name
+                - key: agent.name # set via elastictraceprocessor
+              attributes:
+                - key: span.name
+                - key: event.outcome
+                - key: service.target.type
+                - key: service.target.name
+                - key: span.destination.service.resource
+                - key: metricset.name
+                  default_value: service_destination
+              sum:
+                value: Int(AdjustedCount())
       exporters:
         debug:
           verbosity: basic
-        elasticsearch/apm:
-          endpoints:
-          - ${env:ELASTIC_ENDPOINT}
-          api_key: ${env:ELASTIC_API_KEY}
-          logs_dynamic_index:
-            enabled: true
-          mapping:
-            mode: ecs
```

Comment:
We remove the `elasticsearch/apm` and we are going to use elasticsearch/otel

Just clarifications: Dont  we need
```yaml
          traces_dynamic_index:
            enabled: true
          flush:
            interval: 10s
```

(meanwhile this config was used in elasticsearch/apm)

Reply:
good catch! It actually needs to be enabled, fixed https://github.com/elastic/opentelemetry/pull/27/commits/7c7941e671dbc0a1c3a4f1a864c0460e51e16456